packages/contracts-bedrock/test/L1/L2OutputOracle.t.sol

```diff
@@ -511,4 +489,213 @@ contract L2OutputOracleUpgradeable_Test is CommonTest {
         bytes32 slot21Expected = NextImpl(address(l2OutputOracle)).slot21Init();
         assertEq(slot21Expected, slot21After);
     }
+
+    /// @dev Tests that initialize reverts if the submissionInterval is zero.
+    function test_initialize_submissionInterval_reverts() external {
+        // Reset the initialized field in the 0th storage slot
+        // so that initialize can be called again.
+        vm.store(address(l2OutputOracle), bytes32(uint256(0)), bytes32(uint256(0)));
+
+        uint256 l2BlockTime = deploy.cfg().l2BlockTime();
+        uint256 startingBlockNumber = deploy.cfg().l2OutputOracleStartingBlockNumber();
+        uint256 startingTimestamp = deploy.cfg().l2OutputOracleStartingTimestamp();
+        address proposer = deploy.cfg().l2OutputOracleProposer();
+        address challenger = deploy.cfg().l2OutputOracleChallenger();
+        uint256 finalizationPeriodSeconds = deploy.cfg().finalizationPeriodSeconds();
+
+        vm.expectRevert("L2OutputOracle: submission interval must be greater than 0");
+        l2OutputOracle.initialize({
+            _submissionInterval: 0,
+            _l2BlockTime: l2BlockTime,
```

Comment:
You could inline the calls to `deploy.cfg()...` inline in the constructor to reduce lines of code